docs/running-node.md

```diff
@@ -17,7 +17,7 @@ This document describes in details how to configure different types of DCLedger
     *   The list of alive peers: `persistent_peers.txt` with the following format: `<node id>@<ip:port>,<node2 id>@<ip:port>,...`.
     *   where to get:
         *   If you want to join an existing persistent network (such as Test Net), then look at the [Persistent Chains](../deployment/persistent_chains)
-            folder for a list of available networks. Each subfolder there represents a `<chain-id>` and contains the genesis and persistent_peers files.
+            folder for a list of available networks. Each sub-folder there represents a `<chain-id>` and contains the genesis and persistent_peers files.
```

Comment:
Let's mention that `persistent_peers` may not be available there, and network admins need to be contacted to get it

Reply:
updated